sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -364,3 +364,34 @@ task validatesContainer() {
     }
   }
 }
+
+def tensorRTTests = tasks.create("tensorRTtests") {
+  dependsOn 'installGcpTest'
+  dependsOn ':sdks:python:sdist'
+ doLast {
+  def testOpts = basicPytestOpts
+  def argMap = [
+    "runner": "DataflowRunner",
+    "machine_type":"n1-standard-4",
+    "sdk_container_image": "us.gcr.io/apache-beam-testing/yeandy/apache_beam_tensorrt:latest",
+    "sdk_location": files(configurations.distTarBall.files).singleFile,
+    "project": "apache-beam-testing",
+    "region": "us-central1",
+    "input": "gs://apache-beam-testing-yeandy/tensorrt_image_file_names.txt",
```

Comment:
Can we make make a copy of `gs://apache-beam-testing-yeandy/tensorrt_image_file_names.txt` and put one under `gs://apache-beam-ml/testing/inputs`?